src/app/[locale]/viewer/_components/submodel/carbon-footprint/visualization-components/ProductJourney.tsx

```diff
@@ -118,19 +117,18 @@ function getMarkerLayers(coordinatesPerLifeCyclePhase: AddressPerLifeCyclePhase[
                 ],
             });
 
-            const markerIconName = `MarkerIcon${phase.lifeCyclePhase}`;
-
-            const productJourneyIconName = ProductJourneyIcons[markerIconName];
-
+            const markerIconName = `LocationMarker_${phase.lifeCyclePhase}`;
+            
             return new VectorLayer({
                 source: markerSource,
                 style: new Style({
                     image: new Icon({
                         anchor: [0.5, 1],
-                        src: productJourneyIconName ? productJourneyIconName.src : '',
+                        src: `/LocationMarkers/${markerIconName}.svg`,
```

Comment:
With this changes, --turbo works and is way faster than the regular 'next dev'. The downside is, that we can no longer use the svg?url solution we had before. I tried to find a quick solution but I'm not convinced 😄